acceptance-tests/acceptance_tests_suite_test.go

```diff
@@ -64,3 +68,46 @@ func setupTunnelFromLocalMachineToHAProxy(haproxyInfo haproxyInfo, localPort, ha
 
 	return cancelFunc
 }
+
+func expectTestServer200(resp *http.Response, err error) {
+	Expect(err).NotTo(HaveOccurred())
+	Expect(resp.StatusCode).To(Equal(http.StatusOK))
+	Eventually(gbytes.BufferReader(resp.Body)).Should(gbytes.Say("Hello cloud foundry"))
+}
+
+func expect200(resp *http.Response, err error) {
```

Comment:
why not
```
func expectStatus(resp *http.Response, status int, err error) {
...
```
?

Reply:
Because if it only accepts two arguments you can pass them both directly from the output of another function like this `expect200(httpClient.Get(...))`

with three arguments you would need to write
```
resp, err := httpClient.Get(...)
expectStatus(resp, err, http.StatusOK)
```

because `expectStatus(httpClient.Get(...), http.StatusOK)` is not valid Go

Reply:
Ah I see. Too bad ((a,b),c) does not unfold to (a,b,c)